chemprop/v2/data/utils.py

```diff
@@ -2,65 +2,253 @@
 from typing import Sequence
 
 import numpy as np
-from rdkit.Chem.Scaffolds import MurckoScaffold
 
 
 from chemprop.v2.data.datapoints import MoleculeDatapoint
+from chemprop.v2.data.datasets import MoleculeDataset
+
+from astartes import train_val_test_split, train_test_split
+from astartes.molecules import train_val_test_split_molecules, train_test_split_molecules
+
+import copy
+from logging import Logger
+import pickle
+from random import Random
+from typing import Tuple
+import os
+
+from chemprop.data import log_scaffold_stats
+from chemprop.args import TrainArgs
 
 
 def split_data(
     data: Sequence[MoleculeDatapoint],
     split: str = "random",
-    sizes: tuple[float, float, float] = (0.8, 0.1, 0.1),
-    k: int = 5,
-    fold: int = 0,
-):
+    sizes: Tuple[float, float, float] = (0.8, 0.1, 0.1),
```

Comment:
`sizes: tuple[...`
The current convention in v2 has been to use lowercase types (tuple, list), which was added in Python 3.9
Not a must change, but could be good.

Reply:
The outer types have to be capitalized since they are not built-ins, so I think I will leave this as is

Reply:
After further discussion, we decided to keep it as lowercase tuple. Not backwards compatible with older python versions but consistent with the rest of the v2 code.

Reply:
Bumping that Tuple should be replaced with the built in version and the import removed.